scipy/ndimage/tests/test_interpolation.py

```diff
@@ -851,6 +851,22 @@ def test_affine_transform_multi_d_endianness_with_output_parameter(self):
             result = out if returned is None else returned
             assert_array_almost_equal(result, [1])
 
+
+    def test_affine_transform_output_shape(self):
+        # don't require output_shape when out of a different size is given
+        data = numpy.arange(8, dtype=numpy.float64)
+        out = np.ones((16,))
```

Comment:
`numpy` is used instead of `np` throughout this module (the alias is not imported as `np`--perhaps a really old test module or something?), so looks like a simple fix to get CI green here maybe

Reply:
thanks. I am pretty sure this had been passing, but I must have modified it later and not rerun the test! should be fixed now